tests/client/application_client_test.py

```diff
@@ -5,15 +5,22 @@
 import pyteal as pt
 import pytest
 from algokit_utils import CallConfig, LogicError, MethodHints
+from algosdk import transaction
```

Comment:
We're importing from transaction below, can we just import Transaction from there as well?

Reply:
Good spot!!, done